src/Uno.UI.Runtime.Skia.Tizen/Tizen/TizenUIElementPointersSupport.cs

```diff
@@ -33,13 +33,22 @@ public class TizenUIElementPointersSupport : ICoreWindowExtension
 		private readonly GestureLayer _gestureLayer;
 		private readonly UnoCanvas _canvas;
 
+		public CoreCursor PointerCursor
+		{
+			get => new CoreCursor(CoreCursorType.Arrow, 0);
+			set
```

Comment:
Codacy found an issue: [Use the 'value' parameter in this property set accessor declaration.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6206423)